devo/api/client.py

```diff
@@ -207,9 +231,7 @@ def __init__(self, address=None, auth=None, config=None,
 
         self.auth = auth
         if not address:
-            raise raise_exception(
-                _format_error(ERROR_MSGS['no_endpoint'])
-            )
+            raise raise_exception(ERROR_MSGS['no_endpoint'],400)
```

Comment:
If we don't have address, we cannot create an HTTP request and no HTTP status code can be returned. It will create confussion in developer/use as no HTTP service can have returned any status code. Do not create HTTP status codes. Just returns the one returned by Serrea.